azuredriver/service/vnet_arm.py

```diff
@@ -1,25 +1,31 @@
-
-from azuredriver.location import *
-from azuredriver.model.exceptions import *
-from azuredriver.service.azureresourcemanager import *
+'''Python script for VNET ARM'''
 import logging
-import json
+from azuredriver.service.azureresourcemanager import *
```

Comment:
Wildcard import can be replaced with AzureResourceManager import

Reply:
Done